packages/snap-preact-components/src/components/Atoms/SearchHeader/SearchHeader.tsx

```diff
@@ -131,10 +131,11 @@ export interface SearchHeaderProps extends ComponentProps {
 	controller?: SearchController;
 	queryStore?: SearchQueryStore;
 	paginationStore?: SearchPaginationStore;
+	merchandisingStore?: SearchMerchandisingStore;
 
 	titleText?: string | ((data: data) => string);
 	subTitleText?: string | ((data: data) => string);
```

Comment:
I just realized that "subtitle" is a single word, so we can lose the camelcase here.